### DISCUSSION TITLE
Match all content apart certain tags

### DISCUSSION THREAD

**Discussion by thewebartisan7** [posthtml/posthtml#388] [Q&A] [answered]
Something like with css `{tag: ':not(div)'}` how we can do with match or match helper?

Thanks!

**Comment by thewebartisan7** [comment]
I handle this with array filters, but I would like to know how to do this with posthtml.

```js
  const contentWithoutDiv = content.filter(c => c.tag !== 'div');
```

**Comment by cossssmin** [comment, marked as answer]
Yeah, `:not` isn't supported as a matcher in PostHTML or [posthtml-match-helper](https://github.com/posthtml/posthtml-match-helper). Would be a nice thing to have in the latter so you could do something like `tree.match(matchHelper(':not(div)'), node => { ... })`

**Comment by thewebartisan7** [comment]
Thank You @cossssmin 
Yes something like that would be great.
I was trying to do this like for attribute negation, but no way.
